Microsoft.Toolkit.Uwp.UI.Controls/ColumnResizer/ColumnResizer.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// ColumnResizer is a UI control that add the resizing functionality to a Grid Column.
+    /// </summary>
+    [TemplatePart(Name = RESIZERNAME, Type = typeof(Grid))]
+    public partial class ColumnResizer : Control
```

Comment:
I suggest calling this `GridSplitter` and inheriting from `Thumb` instead. This'll make it match WPF